### PR TITLE
Openstack/compare

### LINKED ISSUE #2443: some desktop/government references out of date
Munich recently decided to ditch Linux, so probably not worth a callout.

CESG doesn't actually exist anymore, the report is nearly four years old, so not 'recent' as mentioned in text 
---
*Reported from: https://www.ubuntu.com/desktop/government*

### PR DESCRIPTION
**Pull request by deadlight**
Added new /openstack/compare page

### QA

Check [the page](https://ubuntu-com-canonical-web-and-design-pr-6995.run.demo.haus/openstack/compare) in-browser against the [copydoc](https://docs.google.com/document/d/1ibpATi-e0rKP3GaLgvzwO_XyFU9ZTsUaCHMwrg96MZM/edit#).

Note: I've made some changes from the original [design](https://github.com/canonical-web-and-design/web-squad/issues/2428) at my discretion since the content had changed considerably and the page was looking a bit dull.

Fixes #2443

### CODE REVIEW

**Comment by deadlight** [comment]
@clagom this looks a bit rough but do you think that I should work to clean this up or should I go the route that @pmahnke [suggested](https://github.com/canonical-web-and-design/web-squad/issues/2428#issuecomment-597509368)

**Comment by clagom** [comment]
I think it works @deadlight, I would just align the + to the first line of text (not centred aligned). If it's ok for @anthonydillon @pmahnke Vanilla wise

File: templates/openstack/compare.html [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+          <td>Per socket-pair</td>
+          <td>Per host + base price for the control plane</td>
+        </tr>
+
+        <tr>
+          <th>Availability of consulting services</th>
+          <td class="p-accordion">
+            <button class="p-accordion__tab" id="tab-consulting" role="tab" aria-controls="content-consulting" aria-expanded="false">Yes</button>
+            <div class="p-accordion__panel" aria-hidden="true" id="content-consulting">
+              <p>Canonical offers consulting services for OpenStack including design and delivery at a fixed price which helps to make deployment costs and timing more predictable.</p>
```
Comment (anthonydillon) [inline templates/openstack/compare.html]: Could you add the links here to see how that looks and works?

**Comment by anthonydillon** [comment]
@clagom good point. I have [filed that issue on Vanilla](https://github.com/canonical-web-and-design/vanilla-framework/issues/2926). You call whether @deadlight implements a fix locally or if we can go live with this and adopt the fix in a future Vanilla release?

**Comment by clagom** [comment]
> @clagom good point. I have [filed that issue on Vanilla](https://github.com/canonical-web-and-design/vanilla-framework/issues/2926). You call whether @deadlight implements a fix locally or if we can go live with this and adopt the fix in a future Vanilla release?

Whatever is easier, not a blocker for the release of this page I think.

**Comment by deadlight** [comment]
@anthonydillon @clagom : How's this?

File: templates/openstack/compare.html [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>Open source, unlike VMware</h2>
+      <p>Charmed OpenStack is fully open source which allows it to benefit from a broader community, faster development process and broader selection of technology choices than those available in VMware’s vRealize platform. Choosing an open source platform avoids vendor lock-in. Charmed OpenStack is also much cheaper than VMware vRealize. Under certain circumstances, migrating from VMware to Charmed OpenStack can reduce costs by 88%.</p>
+      <p><a href="/engage/vmware-to-charmed-openstack">Read the whitepaper</a></p>
```
Comment (anthonydillon) [inline templates/openstack/compare.html]: Can you add that little > thingy

**Comment by clagom** [comment]
Looks good, thanks for the change

**Pull request merged**